ensure compatibility with load paths in ruby 1.9 mode

munkyboy opened the pull request:
fixes issue #12

Commit:
ensure compatibility with load paths in ruby 1.9 mode

fixes issue #12

iconara commented:
Thanks for your interest in Mikka, I'm not exactly actively maintaining it since quite a long time ago, so I'm not really sure what to do about your bug reports and this pull request. I can merge this, but there's a lot of other things that would need fixing too, like it's using an outdated version of Akka at this point, and I don't know how compatible it would be with the later releases.

Do you actively use Mikka, or are you just evaluating it?

munkyboy commented:
I'm just evaluating it as a way to utilize Akka through JRuby.

iconara commented:
Ok, good to know. If it's ok with you to continue evaluating it as it is, then if you decide on using it I wouldn't mind handing over the maintenance to you, if you want. I won't have time merging and pushing up new gem releases, unfortunately.

Commit:
whitespace

Commit:
add support for `ask` pattern

also change Mikka.await_result timeout option

Commit:
add API for `pipe_to` pattern

Commit:
clean up spec

utilizie rspec memoization

Commit:
operate on actor selections like actor refs

Commit:
add specs for supervision

note: this commit also changes the duration argument of the supervision strategy constructors. previously, it required an instance of duration. now, it will simply pass the arg to `Duration.[]`

Commit:
add rake as a dependency

Commit:
add default rake task of running specs

Commit:
provide shorthand for specifying supervision strategy

Commit:
compartmentalize implicit sender support

Commit:
sq with pipe_to support

Commit:
don't expose ActorRef in Mikka namespace

Commit:
minor cleanup of method aliasing

Commit:
add support for using routers

Commit:
use rspec 2.14.x

Commit:
add ActorRef/Selection#ask_and_wait

Commit:
remove deprecated variants of creating Props

Also add support for passing args to the Actor constructor

Commit:
add helper for getting current routees
